source/go.txt

```diff
@@ -19,34 +19,33 @@ Introduction
 
 This is the official MongoDB Go Driver.
 
-- `General documentation for the MongoDB Go Driver <https://godoc.org/go.mongodb.org/mongo-driver/mongo>`__
+- `Get Started <https://www.mongodb.com/blog/post/mongodb-go-driver-tutorial>`__
```

Comment:
Overall I think this is fine, but a reason to not like it is it recommends to use the now out of fashion godep instead of gomod.

Reply:
I only moved this list item and renamed it (from "Get Started with Go"), but can remove it if you think it's not necessary since I added links to "Quick Start Tutorials".

Reply:
Sounds good, I think that’s better if there isn’t something more recent.